intellij-ide.md

```diff
@@ -188,8 +188,22 @@ types of code changes cannot be hot reloaded though:
 For these changes you can fully restart your application, without having to end
 your debugging session:
 
-1. In the Debug panel at the bottom, click the **Full Application Restart button**
-(shown with an icon containing a circular arrow).
+1. Don't click the Stop button; simply re-click the Run button (if in a run
+session) or Debug button (if in a debug session).
```

Comment:
Here (or elsewhere), we'll want to document how to access the full restart functionality. Currently accessed via:

- shift-click on the hot reload button instead performs a full restart
- hitting the 'run' or 'debug' button while an app is currently running will restart it

And perhaps, a brief blurb on the differences between hot reload (patches changes but maintain state) and full restart (restart the isolate running the app, slower but resets the app's state).